### PR TITLE
[PICA-96] claimable amount of rewards

### PR DESCRIPTION
**Pull request by kkast**
https://app.clickup.com/t/20465559/PICA-96
add rpc into staking-rewards pallet for claimable amount of rewards

### CODE REVIEW

**Comment by KaiserKarel** [comment]
Task linked: [PICA-96 Claimable RPC request](https://app.clickup.com/t/20465559/PICA-96)

**Review by KaiserKarel — APPROVED** [review]

File: code/parachain/frame/staking-rewards/rpc/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+#[rpc(client, server)]
+pub trait StakingRewardsApi<BlockHash, AssetId, FinancialNftInstanceId, Balance>
+where
+	AssetId: FromStr + Display + Ord,
+	FinancialNftInstanceId: FromStr + Display,
+	Balance: FromStr + Display,
+{
+	#[method(name = "stakingRewards_getClaimableAmount")]
+	fn get_claimable_amount(
```
Comment (benluelo) [inline code/parachain/frame/staking-rewards/rpc/src/lib.rs:24]: The get_ prefix can be removed

**Review by benluelo — CHANGES_REQUESTED** [review]

File: code/parachain/frame/staking-rewards/rpc/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+use staking_rewards_runtime_api::{ClaimableAmountError, StakingRewardsRuntimeApi};
+
+#[rpc(client, server)]
+pub trait StakingRewardsApi<BlockHash, AssetId, FinancialNftInstanceId, Balance>
+where
+	AssetId: FromStr + Display + Ord,
+	FinancialNftInstanceId: FromStr + Display,
+	Balance: FromStr + Display,
+{
+	#[method(name = "stakingRewards_getClaimableAmount")]
```
Comment (benluelo) [inline code/parachain/frame/staking-rewards/rpc/src/lib.rs:22]: rename this as well

**Review by vimukthi-git — APPROVED** [review]
LGTM :rocket:

**Review by benluelo — APPROVED** [review]

**Review by vimukthi-git — APPROVED** [review]

**Comment by kkast** [comment]
@mergify refresh

**Pull request merged**